snappy/install.go

```diff
@@ -60,6 +60,26 @@ func doUpdate(part Part, flags InstallFlags, meter progress.Meter) error {
 	return nil
 }
 
+// convertToInstalledSnaps takes a slcie of remote snaps that got
+// updated and finds the coresponding local snap parts and returns them
```

Comment:
nit: I think this reads better like:
... updated and returns the corresponding local snap parts.

Also you are missing an 'r' in corresponding.